packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -356,6 +356,16 @@ export function hex2Bytes(hex: string): Buffer {
   return Buffer.from(hex, 'hex');
 }
 
+/**
+ *
+ * @description
```

Comment:
Missing the description and param details in the comment
Also consider adding unit tests for the function